client/state/data-layer/wpcom-http/README.md

```diff
@@ -179,16 +179,22 @@ dispatch( http( {
 ```
 
 If given, the action passed as the second and optional parameter will take the place of all unspecified `onSuccess`, `onFailure`, and `onProgress` responder events.
+
 Because we have a very common pattern when issuing requests there is a built-in helper to hand each action off to the right function based on the (possibly) attached metadata.
 
 ```js
 import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 
 // create request when no meta present, add on success, alert on failure
-dispatchRequest( fetchMenu, addMenuItems, alertFailure );
+dispatchRequest( { initiator: fetchMenu, onSuccess: addMenuItems, onError: alertFailure } );
```

Comment:
can we change to `fetch: fetchMenu`?

Reply:
Sure thing! `initiator` was renamed to `fetch` in latest push.